examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
No, i'm more worried about the evolving menagerie of files that are going to control deployment of components.  Whether cluster up and ansible use the same set or not is a secondary concern, but my primary one is now we have two files which work together to get the job done when in theory at least we could just have one.  Feels like a potential for maintenance mistakes when someone fixes something in the template for cluster up and forgets about the ansible side.  I guess I'd just prefer to see/hear that the ansible installer really needs this as a standalone file before introducing it.


My understanding is @smarterclayton wants to go down a path where components (like logging, metrics, tsb, etc) provide templates for deployment (like you've done here), and ansible customizes those templates (rather than orchestrating the component config out of whole cloth like it does now), primarily by passing parameter values, but perhaps via other means in cases where the template capabilities are insufficient.  But breaking that template into sub-files feels like a step too far to me.

Another alternative would be to inline the entire TSBConfig content into the ConfigMap object and only make the namespaces field a parameter within that inlined content, since that's the only actual thing that's being configured, right?